src/debug_menu.cpp

```diff
@@ -345,10 +345,10 @@ void character_edit_menu()
                     break;
 
                 default:
-                    if( smenu.ret > 3 && smenu.ret < static_cast<int>( vits.size() + 4 ) ) {
-                        auto iter = std::next( vits.begin(), smenu.ret - 4 );
+                    if( smenu.ret > 3 && smenu.ret < static_cast<int>( vits.size() + 5 ) ) {
```

Comment:
But it's better to use a variable instead of all 5s.
```suggestion
                    if( smenu.ret >= 5 && smenu.ret < static_cast<int>( vits.size() + 5 ) ) {
```